pkg/parser/dockerfile.go

```diff
@@ -135,7 +135,10 @@ func (p *Parser) calculateDockerfileHash(
 			newHash.Write(fileContents)
 			hashedAtLeastOneSource = true
 		}); err != nil {
-			return "", dockerfileNode.ParserError(err.Error())
+			p.registerIssuef(api.Issue_WARNING, dockerfileNode.Line, dockerfileNode.Column,
+				"%v %q: %v", ErrFailedToAnalyze, dockerfilePath, err)
+
+			return hex.EncodeToString(oldHash.Sum([]byte{})), nil
```

Comment:
No need to fail fast here: there can be other files that will hash successfully and `hashedAtLeastOneSource` will be `true`.

Reply:
Fixed in b691e712f8c5de75c65e45187f72dfd8972709b8